.azure-pipelines/gpu-tests.yml

```diff
@@ -60,6 +60,8 @@ jobs:
     - bash: |
         python requirements/collect_env_details.py
         python -c "import torch ; mgpu = torch.cuda.device_count() ; assert mgpu >= 2, f'GPU: {mgpu}'"
+        python requirements/_check_avail_strategies.py
+        python requirements/_check_avail_extras.py
```

Comment:
These are already printed with the `pip list` in the previous step (L57)

Reply:
yes, but the `pip list` is never verified, the print here is just to have qazi testing that the import really works...